roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -4,7 +4,7 @@
     path: "{{ etcd_cert_dir }}"
     group: "{{ etcd_cert_group }}"
     state: directory
-    owner: kube
+    owner: {{ kube_owner }}
```

Comment:
Need to add " to pass a lint job as
```
The error appears to be in '/builds/kargo-ci/kubernetes-sigs-kubespray/roles/etcd/tasks/gen_certs_script.yml': line 7, column 13, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
    state: directory
    owner: {{ kube_owner }}
            ^ here
We could be wrong, but this one looks like it might be an issue with
missing quotes. Always quote template expression brackets when they
start a value. For instance:
    with_items:
      - {{ foo }}
Should be written as:
    with_items:
      - "{{ foo }}"
```